notebooks/growth_model_forecast.ipynb

```diff
@@ -7,14 +7,23 @@
    "source": [
```

Comment:
<p>I get a 404 error when I click on the link: <a href="https://openai.com/blog/openai-apihttps://openai.com/blog/openai-api" target="_blank">https://openai.com/blog/openai-apihttps://openai.com/blog/openai-api</a>. Also, I was thinking you can link the notebook as well: <a href="https://github.com/redhat-et/customer-growth-model/blob/update/notebooks/growth_model_generate_data.ipynb" target="_blank">https://github.com/redhat-et/customer-growth-model/blob/update/notebooks/growth_model_generate_data.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>